docker/README.md

```diff
@@ -24,7 +24,13 @@ docker run -it --entrypoint expand-data-corpus.py opensearchstaging/opensearch-b
 # Building a Copy of OpenSearch Benchmark Image
 1. Git clone OpenSearch Benchmark Github repository
 2. Make changes locally
-3. Run `docker build -t <docker image name>:<optional tag> -f ./docker/Dockerfile .`
+3. Run `docker build --build-arg VERSION=<PYPI VERSION> -t <docker image name>:<optional tag> -f ./docker/Dockerfile .`
```

Comment:
Build time is missed here due to the value is coming from the build script in build repo.